test/unittest/tests/wrappers/mock_session.cpp

```diff
@@ -34,3 +41,18 @@ MockSession::buildTestMockSession()
     // Construct an object that will now own the two pointers passed in.
     return std::shared_ptr<MockSession>(new MockSession(sessionImpl, mockConnection));
 }
+
+int
+handleWiredTigerError(
+  WT_EVENT_HANDLER *handler, WT_SESSION *session, int error, const char *message)
+{
+    ((handler_wrap *)handler)->mock_session->add_callback_message(message);
+    return (0);
+}
+
+int
+handleWiredTigerMessage(WT_EVENT_HANDLER *handler, WT_SESSION *session, const char *message)
+{
+    ((handler_wrap *)handler)->mock_session->add_callback_message(message);
```

Comment:
In C++, it's better (and clearer) to use C++ style casts rather than C style ones. For example `static_cast<>`, in this case.

Reply:
I think this needs a reinterpret cast, static wasn't sufficient.

Reply:
You are correct - my mistake. `reinterpret_cast<>` is indeed needer here.